swagger-spec/comments/report_detail.yaml

```diff
@@ -1 +1,93 @@
 # /comments/{comment_id}/reports/{user_id}/
```

Comment:
Note to self: purposely not reviewing since this endpoint is commented out in swagger.yaml ATM.